backend/qemu.pm

```diff
@@ -108,6 +108,9 @@ sub eject_cd {
                 (defined $args->{id} || !defined $args->{device} ? (id => $args->{id} // 'cd0-device') : (device => $args->{device})),
                 force => (!defined $args->{force} || $args->{force} ? Mojo::JSON->true : Mojo::JSON->false)
     }});
+    $self->handle_qmp_command({execute => 'blockdev-remove-medium', arguments => {
+                id => $args->{id} // 'cd0-device'
```

Comment:
I'd avoid repeating the `$args->{id} // 'cd0-device'` part (and rather save it in a variable). Then this invocation likely fits into one line.